src/components/utils/kpi_mapping.py

```diff
@@ -1,6 +1,33 @@
-import pandas as pd
+"""Load kpi mapping."""
```

Comment:
Wondering if this should be a part of one of the 4 data preprocessing notebooks instead of the source package... Making a source file interact with s3 doesn't seem like a very good idea. However, changing this would mean changing text and table curator dependencies. A separate issue maybe, WDYT?